x11-packages/kitty/prepend-prefix-to-etc-and-share-paths.patch

```diff
@@ -0,0 +1,40 @@
+--- a/kitty/cli.py
++++ b/kitty/cli.py
+@@ -1063,7 +1063,7 @@ def parse_args(
+     return ans, parse_cmdline(oc, disabled, ans, args=args)
+ 
+ 
+-SYSTEM_CONF = f'/etc/xdg/{appname}/{appname}.conf'
++SYSTEM_CONF = f'@TERMUX_PREFIX@/etc/xdg/{appname}/{appname}.conf'
+ 
+ 
+ def default_config_paths(conf_paths: Sequence[str]) -> tuple[str, ...]:
+--- a/shell-integration/bash/kitty.bash
++++ b/shell-integration/bash/kitty.bash
+@@ -10,13 +10,13 @@ if [[ -n "$KITTY_BASH_INJECT" ]]; then
+     builtin unset KITTY_SHELL_INTEGRATION  # ensure manual sourcing of this file in bashrc does not have any effect
+     builtin unset KITTY_BASH_INJECT ENV
+     if [[ -z "$HOME" ]]; then HOME=~; fi
+-    if [[ -z "$KITTY_BASH_ETC_LOCATION" ]]; then KITTY_BASH_ETC_LOCATION="/etc"; fi
++    if [[ -z "$KITTY_BASH_ETC_LOCATION" ]]; then KITTY_BASH_ETC_LOCATION="@TERMUX_PREFIX@/etc"; fi
```

Comment:
Yep, that'd do it.
Baffles me that they wouldn't just let the shell take care of it's configuration.